controllers/user/update-email.js

```diff
@@ -77,7 +77,7 @@ router
                         newEmail: username
                     });
                     const updatedUser = await Users.findById(userId);
-                    await sendActivationEmail(req, updatedUser, true);
+                    await sendActivationEmail(req, updatedUser);
```

Comment:
Do we need to re-add the date of activation here too? What happens when this is `null`?

Reply:
Ahh good catch, forgot we sent it there too. Adding it now.